app/assets/src/components/views/projects/projects_view.scss

```diff
@@ -0,0 +1,54 @@
+@import "~styles/themes/colors";
+@import "~styles/themes/typography";
+
+.project {
+  text-align: left;
+  padding: 15px 0;
+
+  &Name {
```

Comment:
Small nit just because I noticed it. I prefer to just use `cs.name` in the JSX and `.name` here, instead of `cs.projectName` and `&Name`. I think it's a little easier to read. And I can CTRL+F for the class and find it easily. If I tried to CTRL+F `projectName`, I wouldn't find it anywhere in the project. Let me know if there's a reason you prefer `&Name`.

Reply:
I like `projectName` in the html code because i find it easier to read in markup language (not a prob in css).
But I think your "searchable" argument is valid and more relevant. So I can use that.